kpi_scan/kpi_repo_list.json5

```diff
@@ -1,36 +1,93 @@
 {
-  "include": [
```

Comment:
The old formatting for this file was much easier to read

Reply:
My editor did that when I saved.